File: recyclerviewjava/src/main/java/com/example/recyclerviewjava/MainActivity.java
package com.example.recyclerviewjava;

import androidx.appcompat.app.AppCompatActivity;
import androidx.recyclerview.widget.LinearLayoutManager;
import androidx.recyclerview.widget.RecyclerView;

import android.os.Bundle;
import android.widget.LinearLayout;

import java.util.ArrayList;

public class MainActivity extends AppCompatActivity {
ArrayList<RecyclerViewItems>recyclerViewItems;
private RecyclerView recyclerView;
private RecyclerView.Adapter adapter;
private RecyclerView.LayoutManager layoutManager;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
recyclerViewItems=new ArrayList<>();
recyclerViewItems.add(new RecyclerViewItems(R.drawable.ic_android_black_24dp,Utils.TEXT_VIEW_1,Utils.TEXT_VIEW_2));
recyclerViewItems.add(new RecyclerViewItems(R.drawable.ic_android_black_24dp,Utils.TEXT_VIEW_1,Utils.TEXT_VIEW_2));
recyclerViewItems.add(new RecyclerViewItems(R.drawable.ic_android_black_24dp,Utils.TEXT_VIEW_1,Utils.TEXT_VIEW_2));
recyclerViewItems.add(new RecyclerViewItems(R.drawable.ic_android_black_24dp,Utils.TEXT_VIEW_1,Utils.TEXT_VIEW_2));
recyclerViewItems.add(new RecyclerViewItems(R.drawable.ic_android_black_24dp,Utils.TEXT_VIEW_1,Utils.TEXT_VIEW_2));
recyclerView=findViewById(R.id.recyclerView);
recyclerView.setHasFixedSize(true);
adapter=new RecyclerViewAdapter(recyclerViewItems);
layoutManager= new LinearLayoutManager(this);
recyclerView.setAdapter(adapter);
recyclerView.setLayoutManager(layoutManager);
    }
}

File: recyclerviewkotlin/src/main/java/com/example/recyclerviewkotlin/MainActivity.kt
package com.example.recyclerviewkotlin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.LinearLayout
import androidx.recyclerview.widget.LinearLayoutManager
import androidx.recyclerview.widget.RecyclerView

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val recyclerView=findViewById(R.id.recivlerView)as RecyclerView
        recyclerView.layoutManager=LinearLayoutManager(this,LinearLayout.VERTICAL,false)
        val user=ArrayList<User>()
        user.add(User("Poxos","Pssfsf"))
        user.add(User("Poxos","Pssfsf"))
        user.add(User("Poxos","Pssfsf"))
        val adapter=CustyomAdapter(user)
        recyclerView.adapter=adapter
    }
}
